api/repositories/namespace_retriever.go

```diff
@@ -82,6 +82,12 @@ var (
 		Resource: "cftasks",
 	}
 
+	RoleBindingGVR = schema.GroupVersionResource{
```

Comment:
I wonder whether we need to register role bindings with the namespace retriever. Initially I though we should, but then I think the repository ended up not using it

Reply:
yes, it turned out it is not needed